OpenOversight/migrations/versions/59e9993c169c_change_faces_to_thumbnails.py

```diff
@@ -0,0 +1,44 @@
+"""change_faces_to_thumbnails
+
+Revision ID: 59e9993c169c
+Revises: bd0398fe4aab
+Create Date: 2018-06-04 19:04:23.524079
+
+"""
+
+from app.models import Face, db
+from app.utils import get_uploaded_cropped_image
+
+
+# revision identifiers, used by Alembic.
+revision = '59e9993c169c'
+down_revision = 'bd0398fe4aab'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    for face in Face.query.all():
+        # import pdb; pdb.set_trace()
+        if face.face_position_x \
```

Comment:
so I noticed while testing this migration in the dev env that the test data does not have the bounding box fields filled in. since those columns haven't actually been required (this was a holdover from the early days when we manually made cutouts of pics before submitting them to the website), it's probably worth doing:

```
try:
    if face.face_position_x \ 
    [...]
except AttributeError:
    pass  # then skip this face
```